test/quick_test/test_images.py

```diff
@@ -0,0 +1,222 @@
+import pytest
+import nibabel as nib
+import nibabel.cmdline.diff
+import numpy as np
+from pathlib import Path
+
+from collections import OrderedDict
+
+from .common import *
+
+from CerebNet.utils.metrics import dice_score
+
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+def load_image(subject_path: Path, image_name: Path):
+    """
+    Load the image data using nibabel.
+
+    Parameters
+    ----------
+    subject_path : Path
+        Path to the subject directory.
+    image_name : Path
+        Name of the image file.
+
+    Returns
+    -------
+    nibabel.nifti1.Nifti1Image
+        Image data.
+    """
+    image_path = subject_path / "mri" / image_name
+    image = nib.load(image_path)
+
+    return image
+
+
+def compute_dice_score(test_data, reference_data, labels):
+    """
+    Compute the dice score for each class.
+
+    Parameters
+    ----------
+    test_data : np.ndarray
+        Test image data.
+    reference_data : np.ndarray
+        Reference image data.
+    labels : np.ndarray
+        Unique labels in the image data.
+
+    Returns
+    -------
+    np.ndarray
+        Dice scores for each class.
+    """
+
+    # Classes
+    num_classes = len(labels)
+
+    dscore = np.zeros(shape=num_classes)
+
+    for idx in range(num_classes):
+        current_label = labels[idx]
+
+        pred = (test_data == current_label).astype(int)
+        gt = (reference_data == current_label).astype(int)
+
+        dscore[idx] = dice_score(pred, gt)
+
+    logger.debug("\nDice score: ", dscore)
+
+    return dscore
+
+
+def compute_mean_square_error(test_data, reference_data):
+    """
+    Compute the mean square error between the test and reference data.
+
+    Parameters
+    ----------
+    test_data : np.ndarray
+        Test image data.
+    reference_data : np.ndarray
+        Reference image data.
+
+    Returns
+    -------
+    float
+        Mean square error.
+    """
+
+    mse = ((test_data - reference_data) ** 2).mean()
+    logger.debug("\nMean square error: ", mse)
+
+    return mse
+
+
+@pytest.mark.parametrize("test_subject", load_test_subjects())
+def test_image_headers(subjects_dir: Path, test_dir: Path, reference_dir: Path, test_subject: Path):
+    """
+    Test the image headers by comparing the headers of the test and reference images.
+
```

Comment:
Probably also add the documentation for the fixtures here.

Reply:
done